skorch/tests/callbacks/test_lr_scheduler.py

```diff
@@ -35,6 +35,41 @@ def test_simulate_lrs_batch_step(self, policy):
         expected = np.array([1, 2, 3, 4, 5, 4, 3, 2, 1, 2, 3])
         assert np.allclose(expected, lrs)
 
+    def test_simulate_lrs_reduced_lr_on_plateau_scalar(self):
+        # Feed a constant, scalar "loss" to the scheduler.
+        lr_sch = LRScheduler(
+            ReduceLROnPlateau, factor=0.1, patience=1,
+        )
+        lrs = lr_sch.simulate(
+            steps=5, initial_lr=1, step_args=0.5
+        )
+        # O = OK epoch
+        # I = intertolerable epoch
+        #
+        # O I I I I   epoch classification
+        # 0 1 2 3 4   number of bad epochs
+        #     * * *   epochs with LR reduction
+        #
+        # note that simulate returns the LRs before the step, not after,
+        # so we're seeing only 4 updated values.
+        assert all(lrs == [1] + [1, 1, 0.1, 0.1])
```

Comment:
Do we see "4 updated values"? Why use `[1] + ...` here but not in the next test?